Lib/ufo2ft/featureWriters/kernFeatureWriter2.py

```diff
@@ -240,6 +271,136 @@ def getKerningPairs(font, side1Classes, side2Classes, glyphSet=None):
                 result.append(KerningPair(side1, side2, value))
         return result
 
+    @staticmethod
+    def getVariableKerningPairs(
```

Comment:
better to put this somewhere it can be shared by both kern writers than copy/paste

Reply:
I'm calling into the newer writer now.